src/analyze/analyze-syscall-filter.c

```diff
@@ -58,15 +58,35 @@ static int load_kernel_syscalls(Set **ret) {
         return 0;
 }
 
+static int syscall_set_add(Set **s, const SyscallFilterSet *set) {
+        int r;
+
+        assert(s);
+
+        if (!set)
+                return 0;
+
+        NULSTR_FOREACH(sc, set->value) {
+                if (sc[0] == '@')
+                        continue;
```

Comment:
Shouldn't this add `sc` recursively?

Reply:
no. this is called from a loop around all groups. so if we follow the groups here we'd add them multiple times, which is pointless.